cmd/servicebroker/main.go

```diff
@@ -22,15 +26,19 @@ import (
 var options struct {
 	broker.Options
 
-	Port    int
-	TLSCert string
-	TLSKey  string
+	Port         int
+	TLSCert      string
+	TLSKey       string
+	Authenticate bool
+	KubeConfig   string
 }
 
 func init() {
 	flag.IntVar(&options.Port, "port", 8005, "use '--port' option to specify the port for broker to listen on")
 	flag.StringVar(&options.TLSCert, "tlsCert", "", "base-64 encoded PEM block to use as the certificate for TLS. If '--tlsCert' is used, then '--tlsKey' must also be used. If '--tlsCert' is not used, then TLS will not be used.")
 	flag.StringVar(&options.TLSKey, "tlsKey", "", "base-64 encoded PEM block to use as the private key matching the TLS certificate. If '--tlsKey' is used, then '--tlsCert' must also be used")
+	flag.BoolVar(&options.Authenticate, "authenticate", false, "option to specify if the broker should validate the bearer auth token with kubernetes")
```

Comment:
We should call this CLI flag `authenticate-k8s-token` or something. CF only currently supports basic auth.